app/lib/common/dialogs/invite_to_room_dialog.dart

```diff
@@ -158,6 +159,7 @@ class _InviteToRoomDialogState extends ConsumerState<InviteToRoomDialog>
     final room = ref.watch(briefRoomItemWithMembershipProvider(roomId));
     final invited =
         ref.watch(roomInvitedMembersProvider(roomId)).valueOrNull ?? [];
+    final joined = ref.watch(membersIdsProvider(roomId)).valueOrNull ?? [];
```

Comment:
may I suggest that we move this watch into the `UserStateButton`? That would remove the flicker of refreshing the entire page when we invited someone or their state changed ...

Reply:
Good point! fixed by 08838aa